concepticondata/conceptlists/Zhou-2020-350.tsv

```diff
@@ -0,0 +1,351 @@
+Zhou-2020-350-262	262	to laugh	笑	1355	LAUGH
+Zhou-2020-350-263	263	to lay (egg)	下（蛋）	2071	LAY (VERB)
+Zhou-2020-350-264	264	to learn	學	504	LEARN
+Zhou-2020-350-265	265	to lend	借出	582	LEND
+Zhou-2020-350-266	266	to lend out	借出		
+Zhou-2020-350-267	267	to lick	舔	319	LICK
+Zhou-2020-350-268	268	to lie down, to lower one's head	低頭、躺	215	LIE DOWN
+Zhou-2020-350-269	269	to lie on one's belly	趴著、孵 ༴ 'to lie down'		
+Zhou-2020-350-270	270	to light up	點燃	737	LIGHT (IGNITE)
+Zhou-2020-350-271	271	to love	愛	923	LOVE
+Zhou-2020-350-272	272	to marry (receive)	娶、取	2164	MARRY (AS MAN)
+Zhou-2020-350-273	273	to measure	量	1506	MEASURE
+Zhou-2020-350-274	274	to need, to owe	需要、欠	2339	NEED (SOMETHING)
+Zhou-2020-350-275	275	to offer (to god)	敬		
+Zhou-2020-350-276	276	to open	打開	1156	OPEN
+Zhou-2020-350-277	277	to paste	貼		
+Zhou-2020-350-278	278	to permeate	滲入		
+Zhou-2020-350-279	279	to pestle	碓	225	PESTLE
+Zhou-2020-350-280	280	to pick (flower)	摘	2148	PICK
+Zhou-2020-350-281	281	to pick up	撿	371	PICK UP
+Zhou-2020-350-282	282	to plant, to stick in	插、栽	1486	PLANT (SOMETHING)
+Zhou-2020-350-283	283	to pound	舂	354	POUND
+Zhou-2020-350-284	284	to press	壓	1913	PRESS
+Zhou-2020-350-285	285	to puke	嘔吐	1278	VOMIT
+Zhou-2020-350-286	286	to pull	拔	1455	PULL
+Zhou-2020-350-287	287	to rain	下（雨）	1253	RAIN (RAINING)
+Zhou-2020-350-288	288	to release	放	1095	LET GO OR SET FREE
+Zhou-2020-350-289	289	to rest	休息	168	REST
+Zhou-2020-350-290	290	to rise (water)	漲		
+Zhou-2020-350-291	291	to roast	烤	1962	ROAST (SOMETHING)
+Zhou-2020-350-292	292	to roll	滾	827	ROLL
+Zhou-2020-350-293	293	to scrape	刮	1767	SCRAPE
+Zhou-2020-350-294	294	to see	看見	1409	SEE
+Zhou-2020-350-295	295	to send away	送（走）		
+Zhou-2020-350-296	296	to sew	縫	1457	SEW
+Zhou-2020-350-297	297	to sit	坐	1649	SIT DOWN
+Zhou-2020-350-298	298	to snow	下（雪）	2234	FALL (OF SNOW AND RAIN)
+Zhou-2020-350-299	299	to soak	浸泡	2069	SOAK (VERB)
+Zhou-2020-350-300	300	to split	劈	3151	CUT (WITH AXE)
+Zhou-2020-350-301	301	to spread out	鋪開	38	SPREAD OUT
+Zhou-2020-350-302	302	to squat	蹲	296	CROUCH
+Zhou-2020-350-303	303	to stand	站	1442	STAND
+Zhou-2020-350-304	304	to steal	偷	713	STEAL
+Zhou-2020-350-305	305	to steam	蒸	2067	STEAM (VERB)
+Zhou-2020-350-306	306	to step on	踩	575	TREAD
+Zhou-2020-350-307	307	to stink	臭	42	STINKING
+Zhou-2020-350-308	308	to store up	收藏	3655	STORE UP
+Zhou-2020-350-309	309	to swallow	吞	897	SWALLOW
+Zhou-2020-350-310	310	to sweep	掃	230	SWEEP
+Zhou-2020-350-311	311	to swing	甩	2871	SWING (MOVEMENT)
+Zhou-2020-350-312	312	to think	想	2271	THINK
+Zhou-2020-350-313	313	to throw	扔	1456	THROW
+Zhou-2020-350-314	314	to tie together	捆		
+Zhou-2020-350-315	315	to tie up	綁	1917	TIE
+Zhou-2020-350-316	316	to touch	摸	1892	TOUCH
+Zhou-2020-350-317	317	to understand, to know	知道	1536	UNDERSTAND
+Zhou-2020-350-318	318	to wait	等待	1590	WAIT (FOR)
+Zhou-2020-350-319	319	to walk	走	1443	WALK
+Zhou-2020-350-320	320	to wash	洗	1453	WASH
+Zhou-2020-350-321	321	to wash	淘洗	1453	WASH
```

Comment:
I'd unmap since the above gloss seems to cover the concept.

Reply:
Yes, the Chinese gloss is more specific. I support unmapping.